src/test/javascript/spec/component/plagiarism/plagiarism-split-view.component.spec.ts

```diff
@@ -128,14 +155,59 @@ describe('Plagiarism Split View Component', () => {
         expect(textSubmissionB).toEqual(submissionB);
     });
 
-    it('parses text matches', () => {
+    it('should parse text matches', () => {
         jest.spyOn(comp, 'mapMatchesToElements').mockReturnValue(new Map());
 
-        const matches: PlagiarismMatch[] = [];
+        const matches: PlagiarismMatch[] = [
+            { startA: 0, startB: 0, length: 5 },
+            { startA: 10, startB: 10, length: 20 },
+        ];
         comp.parseTextMatches({ submissionA, submissionB, matches } as PlagiarismComparison<TextSubmissionElement>);
 
-        expect(comp.matchesA).toBeDefined();
-        expect(comp.matchesB).toBeDefined();
         expect(comp.mapMatchesToElements).toHaveBeenCalledTimes(2);
+        expect(comp.mapMatchesToElements).toHaveBeenCalledWith(
```

Comment:
Please use `.toHaveBeenNthCalledWith` as you can pass there the index of the call.

Reply:
@julian-christl Thanks, I added that now.